### PR TITLE
Prepare for 1.1 release (SHOOP-987)

### PR DESCRIPTION
**Pull request by suutari-ai**
Update change log and add release instructions.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,6 +15,28 @@
 LONG_DESCRIPTION_FILE = os.path.join(TOPDIR, 'README.rst')
 VERSION_FILE = os.path.join(TOPDIR, 'shoop', '_version.py')
 
+# Release instructions
```
Comment (akx) [inline setup.py]: I don't think these belong in a comment in setup.py. Can you make a Real, Actual Document out of these -- maybe .rst in `doc/`, even?

File: ChangeLog.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
+Shoop Change Log
```
Comment (akx) [inline ChangeLog.rst]: I guess the changelog could live in `doc/` too? Would make it easily browseable via RTFD once it unbreaks.

Comment (suutari-ai) [inline ChangeLog.rst]: It's kind of "standard" that ChangeLog is in repository root. Maybe we could symlink it to doc though, but let's check that before the next release. OK?

File: setup.py (hunk 1 above)
Comment (suutari-ai) [inline setup.py]: You only need these when releasing. It's just cruft for those who don't do releases.

On the other hand, when releasing, the first thing you try to accomplish, is to update the version number, which you'll find in setup.py. That's why it's the most visible place for instructions like this. I can move them to doc and add a "link" though, but I'm a bit scared that then nobody follows the "link".

**Comment by akx** [comment]
Let's fix those minor issues later.

**Comment by suutari-ai** [comment]
Thanks, I'll create an issue to our backlog.

**Pull request merged**